src/components/Footer/Footer.js

```diff
@@ -87,11 +88,11 @@ const Footer = ({
             ) : logoExtended ? (
               logoExtended
             ) : (
-              <Icon
-                icon={iconWfpLogoExtendedEn}
-                description="WFP"
-                className="wfp--footer__cta-logo"
-              />
+              <img
```

Comment:
Why do you use the image in this case, not the icon?

Reply:
I couldn't find WFP standard logo in the icons section so I reverted to using the logo

Reply:
We can add the logo to the icon library. Do you have access to https://github.com/wfp/carbon-icons?

Reply:
@Utzel-Butzel not sure I have access, can you please grant me access?